bower.json

```diff
@@ -12,31 +12,29 @@
     "output"
   ],
   "dependencies": {
-    "purescript-arrays": "^4.2.2",
-    "purescript-bigints": "^3.4.0",
-    "purescript-console": "^3.0.0",
-    "purescript-datetime": "^3.4.1",
-    "purescript-eff": "^3.1.0",
-    "purescript-either": "^3.1.0",
-    "purescript-enums": "^3.2.1",
-    "purescript-foldable-traversable": "^3.7.1",
-    "purescript-formatters": "^3.0.1",
-    "purescript-integers": "^3.2.0",
-    "purescript-js-date": "^4.0.0",
-    "purescript-lists": "^4.12.0",
-    "purescript-maybe": "^3.0.0",
-    "purescript-newtype": "^2.0.0",
-    "purescript-prelude": "^3.1.1",
-    "purescript-strings": "^3.5.0",
-    "purescript-tuples": "^4.1.0",
-    "purescript-unicode": "^3.0.2",
-    "purescript-profunctor": "^3.2.0",
-    "purescript-numbers": "^5.0.0",
-    "purescript-decimals": "^3.4.0"
+    "purescript-arrays": "^5.0.0",
+    "purescript-bigints": "^4.0.0",
+    "purescript-console": "^4.1.0",
+    "purescript-datetime": "^4.0.0",
+    "purescript-either": "^4.0.0",
+    "purescript-enums": "^4.0.0",
+    "purescript-foldable-traversable": "^4.0.0",
+    "purescript-formatters": "slamdata/purescript-formatters#ee2a7e93dd124a71e188a75b002289bfffe4618e",
```

Comment:
We should wait on this.

Reply:
https://github.com/slamdata/purescript-formatters/pull/48

Reply:
This was merged, so we can change this to v4.0.0.